cloudify_cli/config/parser_config.py

```diff
@@ -244,26 +246,27 @@ def auto_generate_ids_argument():
 
 def parser_config():
     return {
-        'description': 'Manages Cloudify in different Cloud Environments',
+        'description': 'A Command Line Interface Cloudify',
```

Comment:
it was supposed to be '...for Cloudify' .. but I'm fine with what you're proposing